reservia.css

```diff
@@ -1,53 +1,675 @@
+    background-color: #F2F2F2;
+    border-radius: 40px;
+    width: 42px;
+    height: 29px;
+    text-align: center;
+    padding-top: 12px;
+    margin-left: -4px;   
+}
+
+#coeur
+{
+    border: 1px #F2F2F2 solid;
+    border-left: none;
+    border-radius: 17px 20px 20px 17px;
+    height: 28px;
+    width: 137px;
+    text-align: right;
+    margin-top: -9px;
+    padding-top: 13px;
+    padding-right: 15px;
+    margin-left: -100px; 
+    font-family: "Raleway bold";
+    display: flex;
+}
+
+.coeur
+{
+    color: #0065FC;
+    margin-top: -14px;
+    border: 1px #F2F2F2 solid;
+    background-color: #F2F2F2;
+    border-radius: 40px;
+    width: 42px;
+    height: 29px;
+    text-align: center;
+    padding-top: 12px;
+    margin-left: -4px;
+}
+
+#dog
+{
+    border: 1px #F2F2F2 solid;
+    border-left: none;
+    border-radius: 17px 20px 20px 17px;
+    height: 28px;
+    width: 182px;
+    text-align: right;
+    margin-top: -9px;
+    padding-top: 13px;
+    padding-right: 15px;
+    margin-left: -100px;
+    padding-left: 4px;
+    font-family: "Raleway bold";
+    display: flex;
+}
+
+.dog
+{
+    color: #0065FC;
+    margin-top: -14px;
+    border: 1px #F2F2F2 solid;
+    background-color: #F2F2F2;
+    border-radius: 40px;
+    width: 42px;
+    height: 29px;
+    text-align: center;
+    padding-top: 12px;
+    margin-left: -9px;
+}
+
+#eco div,
+#fam div,
+#coeur div,
+#dog div
+{
+    margin-left: 5px;
+}
+
+#infor
+{
+    border: 1px #F2F2F2 solid;
+    width: 19px;
+    border-radius: 40px;
+    text-align: center;
+    color: #0065FC;
+    margin-left: 15px;
+    margin-top: 24px;
+}
+
+#inform
+{
+    display: flex;
+}
+
+#informa
+{
+    margin-top: 26px;
+    margin-left: 8px;
+}
+
+/*_______________________________________________________________________________________________________________________________________________________________________*/
+
+
+                                                                                                                    /*CSS article*/
+
+
+#hébergement
+{
+    font-size: 18px;
+    margin-left: 40px;
+    margin-top: 50px;
+    margin-bottom: 50px;
+}
+
+
+.hebergements
+{
+    display: flex;
+}
+
+ul
+{
+    list-style-type: none;
+    justify-content: space-around;
+}
+
+.noms
+{
+    margin-left: -35px;
+    font-size: 14px;
+    font-weight: bold; 
+}
+
+.images
+{
+    width: 225px;
+    height: 105px;
+    border-radius: 10px 10px 0px 0px;
+    margin-top: -16px;
+    margin-left: -40px;
+}
+
+#hebergements
+{
+    margin-left: -15px;
+}
+
+.nuits
+{
+    font-size: 12px;
+    margin-left: -35px;
+}
+
+#afficherplus
+{
+    margin-top: 60px;
+    margin-left: 40px;
+    font-size: 14px;
+    font-weight: bold;
+}
+
+.rating
+{
+    color: #F2F2F2;
+    font-size: 11px;
+    margin-left: 4px;
+    margin-top: -15px;
+}
+
+.stars
+{
+    float: left;
+}
+
+.stars2
+{
+    margin-right: -410px;
+}
+
+.stars3
+{
+    margin-right: -125px;
+}
+
+.stars i,
```

Comment:
as a example of why you should delete that file, here you are redefining the style of the stars and if your import of this file in the html where to be the last one it would overidden your other file where you also define style for your style.

Reply:
Yes I adapted the site without reservia.css, I deleted this file

Reply:
nope still there on your remote git repository. https://devconnected.com/how-to-delete-file-on-git/

Reply:
perfect